### PR TITLE
Fix golint issues and move RemoveAllOneFilesystem

### PR DESCRIPTION
**Pull request by hvaara**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
This PR fixes golint issues in `pkg/util/removeall`

**Which issue(s) this PR fixes**:
Ref #68026

**Special notes for your reviewer**:
These changes was originally introduced by @chrischdi in #69761

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Review by mattjmcnaughton — CHANGES_REQUESTED** [review]
/ok-to-test

Thanks for your pr! I suggested a different approach that I feel may be a little cleaner. Let me know what you think :)

File: pkg/util/removeall/removeall.go [hunk 1]
```diff
@@ -25,14 +25,14 @@ import (
 	"k8s.io/kubernetes/pkg/util/mount"
 )
 
-// RemoveAllOneFilesystem removes path and any children it contains.
+// OneFilesystem removes path and any children it contains.
```
[the remark this answers is not in the document]
Comment (hvaara) [reply pkg/util/removeall/removeall.go]: Great idea! I'll do this

[the remark this answers is not in the document]
Comment (hvaara) [reply pkg/util/removeall/removeall.go]: @mattjmcnaughton I've pushed the changes. Thanks a lot for the suggestion - I think it's much cleaner 😄

**Comment by hvaara** [comment]
Thanks again @mattjmcnaughton! Could you PTAL?

/retitle Fix golint issues and move RemoveAllOneFilesystem

**Review by mattjmcnaughton — APPROVED** [review]
/lgtm

+1 this is cleaner :) Thanks!

**Comment by hvaara** [comment]
`mount` has been moved out of tree. Will rebase and fixing conflicts

**Comment by hvaara** [comment]
Will have another go at this soon
/remove-lifecycle stale

**Comment by SergeyKanzhelev** [comment]
@hvaara do you want to rebase PR? If not - I will be happy to take it over.

**Pull request closed without merging**